lib/matplotlib/__init__.py

```diff
@@ -1074,11 +1074,16 @@ def use(backend, *, force=True):
 
         or a string of the form: ``module://my.module.name``.
 
+        Switching to an interactive backend is impossible if an unrelated
```

Comment:
Would slightly prefer: "impossible" -> "not supported" (or maybe "not possible"). But could also be merged as is.

Reply:
Switched to "not possible".  It's really close to impossible because the GUI toolkits globally set the inputhook...

Reply:
Agreed. (But you don't want to discuss that with the end-user.)